js/main.js

```diff
@@ -8,6 +8,17 @@
 
 $(document).ready(function(){
 
+    if ($(window.location.hash)) {
+        topPage = $(window.location.hash);
+        if (!topPage.is(":visible")) {
+            topPage = $("#repositories");
+        }
+
```

Comment:
Would you mind adding the code that update the highlighted menu item (see the "SCROLL TO" section below) ?

Reply:
Thanks! I had completely missed that.